### PR TITLE
Set Brownout Voltage

### PR DESCRIPTION
**Pull request by itsmesp8343**
set brownout voltage to 5

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -29,6 +30,7 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+    RobotController.setBrownoutVoltage(5);
```
Comment (avakimc) [inline src/main/java/frc/robot/Robot.java]: put 5.0 in cal/constants

**Review by avakimc — CHANGES_REQUESTED** [review]
left change

**Review by avakimc — APPROVED** [review]
i think this looks good

**Review by balghane — APPROVED** [review]

**Pull request merged**